### PR TITLE
update service page

### PR DESCRIPTION
**Pull request by AA666666**

### CODE REVIEW

File: client/src/listings/Listings.js [hunk 1]
```diff
@@ -123,27 +124,32 @@ function Listings(props) {
...
+                            ))
+                          ) : (
+                            <></>
+                          )}
+                        </optgroup>
+                      ))
+                    ) : (
+                      <option value='' />
+                    )}
+                  </Select>
```
Comment (JadenBalogh) [inline client/src/listings/Listings.js]: This dropdown looks nice but doesn't actually do anything. Did you forget to hook up the functionality for updating the search results?

File: client/src/profile/ProfileField.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          <Grid className={classes.divider} item xs={9} align='left'>
+            <Typography variant='body2'>{`${details}`}</Typography>
+          </Grid>
+        </Grid>
+      ) : (
+        <></>
+      )}
+    </Grid>
+  );
+}
```
Comment (JadenBalogh) [inline client/src/profile/ProfileField.js:33]: Same as my other review: this code does not appear to be used anywhere, and is a duplicate of another file. Why was it added?

**Review by JadenBalogh — CHANGES_REQUESTED** [review]
Dropdown currently doesn't do anything. See comments.

File: client/src/profile/ProfileField.js (hunk 2 above)
Comment (AA666666) [reply client/src/profile/ProfileField.js:33]: I added it accidentally

File: client/src/listings/Listings.js (hunk 1 above)
Comment (AA666666) [reply client/src/listings/Listings.js]: I thought I only need to do the static one, I will update it

**Review by aKrisnanto — COMMENTED** [review]
This doesn't seem to function properly

**Pull request closed without merging**